### ISSUE TITLE
[Bug]flow chart  clipped

### ISSUE THREAD

**Issue by sospartan** [HappenApps/Quiver#408]
flow chart is not fully display at the bottom :
<img width="570" alt="screen shot 2016-02-04 at 11 48 08 am" src="https://cloud.githubusercontent.com/assets/115201/12805330/6d937e52-cb35-11e5-9b58-462dbcd11a47.png">

my content is :  

```
st=>start: s
e=>end: finish
send=>operation: send
rece=>operation: receive
dl=>operation: download
noti=>operation: notify
hasnew=>condition: new updates?
hasnew2=>condition: more new updates?
unzip=>inputoutput: verify checksum，upzip
replace=>inputoutput: update index,remove old version
checksum=>condition: verify checksum


st->send->rece->hasnew
dl->checksum
checksum(yes)->unzip
checksum(no)->hasnew2
unzip->replace->hasnew2
hasnew(no)->e
hasnew(yes)->dl
hasnew2(yes)->dl
hasnew2(no)->e
```

**Comment by sospartan** [comment]
ping @ylian

**Comment by ylian** [comment]
It seems that flowchart.js fixed this bug in its new release: https://github.com/adrai/flowchart.js

Here is a build if you like to try: https://dl.dropboxusercontent.com/u/3445731/Quiver%20%28Build%2079%29.zip

If it works okay, I will put this in 3.0.2.

**Comment by sospartan** [comment]
@ylian I've test this build, problem solved. :+1:

**Comment by ylian** [comment]
Fixed in 3.0.2.
